engine/lua/cloe/schedule.lua

```diff
@@ -18,6 +18,8 @@
 
 local cloe = cloe or {}
 
+local lust = require 'lust'
```

Comment:
```suggestion
local lust = require("lust")
```

- [ ] Move this to the function where it is used.

Except that best practice in Lua is to only require something at the place you need it.
So not here, but in the function where you are using it, such as `test_fixture` or `schedule_test`.